build/cmake_deps.toml

```diff
@@ -158,6 +158,7 @@ excludes = [
 deps = [
   "xnnpack_backend",
   "portable_kernels",
+  "quantized_kernels",
```

Comment:
Oh yeah good catch

Reply:
I'm still not entirely sure why quantized kernels are needed for `xnn_executor_runner`...

Reply:
Removing this line works for me: https://github.com/pytorch/executorch/blob/main/.ci/scripts/test.sh#L81